op-service/sources/l1_beacon_client.go

```diff
@@ -140,7 +140,7 @@ func (cl *BeaconHTTPClient) BeaconBlobSideCars(ctx context.Context, fetchAllSide
 	}
 
 	if len(indices) > 0 {
-		return eth.APIGetBlobSidecarsResponse{}, fmt.Errorf("#returned blobs(%d) != #requested blobs(%d)", len(hashes)-len(indices), len(hashes))
+		return eth.APIGetBlobSidecarsResponse{}, fmt.Errorf("%s/%s?%s #returned blobs(%d) != #requested blobs(%d)", cl.cl.Endpoint(), reqPath, reqQuery.Encode(), len(hashes)-len(indices), len(hashes))
```

Comment:
This may leak secrets to logs if the api key is in the url path

Reply:
This is a good flag.
We are already logging the full URL path if the endpoint cannot be reached. So we're not really introducing sensitive data here into the logs that we aren't already sometimes showing in logs.
I also think that this is not a concern in CIs where the endpoints would be local lh instances or so, so local Docker endpoints. And in production environments, logs should also already be treated as containing sensitive data.
So I think it's fine to add. But we'll proceed with internal debugging and a dev image from this PR first.

Reply:
I do not think that its normal to have secrets in logs, production or not. A quick google search agrees with this